draft-js-emoji-plugin/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "draft-js-emoji-plugin",
+  "version": "0.0.0",
```

Comment:
i noticed we actually need to compile it with webpack and export the css, otherwise the css import is still in there and it won't work without css modules